cyclonedx_py/client.py

```diff
@@ -251,7 +258,14 @@ def _error_and_exit(message: str, exit_code: int = 1) -> None:
 
     def _get_input_parser(self) -> BaseParser:
         if self._arguments.input_from_environment:
-            return EnvironmentParser(use_purl_bom_ref=self._arguments.use_purl_bom_ref)
+            input_data_fh = self._arguments.input_source
+            with input_data_fh:
+                input_data = input_data_fh.read()
+                input_data_fh.close()
```

Comment:
:x: 
`input_data_fh.close()` not needed, as it will happen automatically on exit of the `with` block